### PR TITLE
Proofread and remove all trailing spaces

### PR DESCRIPTION
**Pull request by ght**
I realized too late that I should have made the commits in the opposite order. If you do not like all the whitespace changes, just let me know and I can resubmit without them.

Also I didn't rebuild the `.html` files, sorry. Should the auto generated files even be under version control? Only makes sense for the gh-pages branch as far as I can tell. (Perhaps add a `CONTRIBUTING.md` file telling people what files they should rebuild for PRs, and how to do so)

### CODE REVIEW

File: webgl/lessons/webgl-2d-matrices.md [hunk 1]
```diff
@@ -364,10 +364,10 @@ We also removed the code that set the resolution. With this last step we've gone
 I hope these posts have helped demystify matrix math. <a href="webgl-3d-orthographic.html">I'll move on to 3D next</a>. In 3D matrix math follows the same principles and usage. I started with 2D to hopefully keep it simple to understand.
 
 <div class="webgl_bottombar">
-<h3>What are <code>clientWidth</code> and <code>clientHeight</code>?</h3>
```
Comment (greggman) [inline webgl/lessons/webgl-2d-matrices.md]: markdown doesn't parse inside HTML. The fact that all of this is wrapped with `<div class="webgl_bottombar">` means everything inside is just copied verbatim. That means your backticks won't get converted into `<code>` blocks.

**Comment by greggman** [comment]
Thanks for this. I wonder if maybe I abuse markdown blockquoting for those webgl_bottombar callouts. That would let the markdown still work.

**Comment by ght** [comment]
I added a new commit reverting those changes. Let me know if there are other issues.

btw:
I was considering doing another pass and "markdownifying" all the existing lessons a bit, any objections or advice on that?

I could also draft a `CONTRIBUTING.md` with a small style guide paragraph, if it should be "x axis" or X axis" or "X-axis" or "xAxis", and things like that, so there is a place to look for consistent rules.

I only recently started learning webgl in depth, so I'll probably go through the existing lessons again anyways, after that I'll go through the todo list and write up some additional ones.

**Comment by greggman** [comment]
I pulled down the patches and manually merged. It's not common to revert something that hasn't actually been committed to master. Generally those commits would be squashed.

**Comment by ght** [comment]
Ok, sorry for the extra work, I'll keep that in mind for the future.

**Pull request closed without merging**